private/js/base.js

```diff
@@ -7,11 +7,14 @@
 // import 'bootstrap/js/dist/alert'
 import ContextGroup from 'components/context-group';
 import PreviewGenerator from 'components/preview-generator';
-import CardGroup from 'components/card-group'
+import ButtonGroup from 'components/button-group';
 
 
 $(() => {
-    // general color context
+    const static_url = $('.djangocms-bootstrap4').data().static;
```

Comment:
what's up with the snake case

Reply:
can be `const { static } = $('...').data();` and then `new Stuff({ static, other: 'options' })`